CHANGES.md

```diff
@@ -6,6 +6,7 @@
 
 - Fixed a crash in the 3D Tiles Feature Styling sandcastle that occurred when using `ModelExperimental`. [#10514](https://github.com/CesiumGS/cesium/pull/10514)
 - Fixed improper handling of double-sided materials in `ModelExperimental`. [#10507](https://github.com/CesiumGS/cesium/pull/10507)
+- Set the behavior of blurring the active element when interacting with the canvas to be configurable. [#10518](https://github.com/CesiumGS/cesium/pull/10518)
```

Comment:
I think this should technically go in a new `##### Additions :tada:` section, and mention the new option by name.

Maybe something like:

```suggestion
- Added optional `blurActiveElementOnCanvasFocus` option to set the behavior of blurring the active element when interacting with the canvas. [#10518](https://github.com/CesiumGS/cesium/pull/10518)
```